smac/facade/smac_facade.py

```diff
@@ -128,7 +128,14 @@ def __init__(self,
         aggregate_func = average_cost
 
         self.scenario = scenario
-        self.output_dir = create_output_directory(scenario, run_id)
+        self.output_dir = ""
+        if not restore_incumbent:
+            self.output_dir = create_output_directory(scenario, run_id)
+        elif scenario.output_dir is not None:
+            # output-dir has been created in CLI because we are restoring.
```

Comment:
I don't understand that comment.

Reply:
Could you maybe replace the comment by something similar to your more lengthy explanation? It is easier to understand.